app/controllers/match_pages_controller.rb

```diff
@@ -0,0 +1,21 @@
+class MatchPagesController < ApplicationController
+  include MatchPagesHelper
+
+  after_action :random_user, only: %i(create)
+  before_action :get_users, :random_user, :logged_in_user
+  before_action :find_user, only: %i(create)
+
+  def create
+    current_user.like @user
+  end
+
+  private
+
+  def get_users
+    @users = User.all
+  end
+
+  def random_user
+    @pagy_user = @users.sample(10)
```

Comment:
```suggestion
    @users = @users.sample(10)
```
đặt tên biến có nghĩa + thêm s nếu nó là số nhiều e nhé

Reply:
em da fix